From f35d6ee76c743cc8c23deb5a7aa786a52585059a Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Wed, 19 Feb 2020 10:07:24 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399pro-npu: adjust npu opp-table

NPU will hang up sometimes with default opp-table:
[ 7710.595423] [galcore]: GPU[0] hang, automatic recovery.
[ 7710.600735] [galcore]: recovery done
[ 7790.723475] [galcore]: GPU[0] hang, automatic recovery.
[ 7790.728833] [galcore]: recovery done

Change-Id: I4524c2294f1c9d876d54e6cda0a1a49185dbacd8
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi b/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
index 3753f1f95e45..da06ae614474 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
@@ -623,10 +623,10 @@
 		};
 		opp-792000000 {
 			opp-hz = /bits/ 64 <792000000>;
-			opp-microvolt = <825000 825000 880000>;
-			opp-microvolt-L0 = <825000 825000 880000>;
-			opp-microvolt-L1 = <800000 800000 880000>;
-			opp-microvolt-L2 = <775000 775000 880000>;
+			opp-microvolt = <850000 850000 880000>;
+			opp-microvolt-L0 = <850000 850000 880000>;
+			opp-microvolt-L1 = <825000 825000 880000>;
+			opp-microvolt-L2 = <800000 800000 880000>;
 		};
 	};
 
-- 
2.35.3

